landoui/static/src/css/components/Footer.scss

```diff
@@ -0,0 +1,3 @@
+.Footer {
```

Comment:
Let's keep selectors as lowercase, especially if we're using `-` to separate selector words instead of camelcase.